terraform/deployments/ecr/main.tf

```
@@ -45,6 +45,7 @@ locals {
     "statsd",
     "govuk-terraform",
     "search-api-learn-to-rank",
+    "content-store-postgresql-branch"
```

tiniest of nits: let's add the defensive trailing comma, just be be nice to the next person who edits this :)
```suggestion
    "content-store-postgresql-branch",
```